_includes/sections/hero.html

```diff
@@ -0,0 +1,39 @@
+{% comment %} 
+<!--
+  Hero Section
+
+  This partial is included on any page that has a "hero" layout with a full bg
+  picture and text/video in a 2 column layout.  Currently on the Home page.
+  
+--> 
+{% endcomment %}
+
+{% assign heros = site.sections|where:"type","hero"|where:"title",include.title %}
+
+{% for hero in heros %}
+
+<section id="slider" class="slider-parallax full-screen dark">
```

Comment:
Can't have an `id` in a for loop.

Reply:
As we discussed in our review meeting, and after trying to remove the id, this did break the js incorporated for the hero.  I don't think it is likely to have 2 heros on one page so I don't think this should be a showstopper.  I think we could add a future improvement task for cleaning up the js more though.  I did what I could with it for purposes of the refactor, but there is still more to untangle when it comes to the js due to the theme.